go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/rancher-sandbox/rancher-turtles
 
-go 1.20
+go 1.22
```

Comment:
you will have problems with merging it to Rancher, Rancher is still based on 1.20 and there are no plans to update it

Reply:
Luckily Rancher doesn't use this directly and we have no direct reliance on Rancher packages :)